tutorials/mesh_to_fuel.md

```diff
@@ -0,0 +1,185 @@
+
+### Thumbnails
+
+Thumbnails are `.png` files named `1` through `5`.
+Each contains a view of the model from a different angle to be displayed on the image carousel on its "Fuel Model Info" page.
+
+### model.config
+
+The `model.config` file is required in the root of the model directory.
+It contains meta data about the model in XML format.
+You have to create this file and add it to the model directory.
+
+Here is the Electrical Box's `model.config`:
+
+```xml
+<?xml version='1.0'?>
+<model>
+  <name>Electrical Box</name>
+  <version>1.0</version>
+  <sdf version='1.6'>model.sdf</sdf>
+
+  <description>
+    Generated with mesh_to_sdf
+    https://bitbucket.org/osrf/mesh_to_sdf
+  </description>
+</model>
+```
+
+Customize the `name` and `description` tags to fit your model.
+The `sdf version` should ideally be the latest version.
+Inside the `sdf` tags is the name of the SDF file describing your model.
+
+### model.sdf
+
+The Electrical Box's `model.sdf` file is as follows:
+
+```xml
+<?xml version='1.0' ?>
+<sdf version='1.6'>
+  <model name='Electrical Box'>
+    <static>true</static>
+    <link name='link'>
+      <pose>0 0 0 1.57 3.14159 0</pose>
+      <collision name='collision'>
+        <geometry>
+          <mesh>
+            <uri>model://Electrical Box/meshes/Electrical Box.dae</uri>
+          </mesh>
+        </geometry>
+      </collision>
+      <visual name='visual'>
+        <geometry>
+          <mesh>
+            <uri>model://Electrical Box/meshes/Electrical Box.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+    </link>
+  </model>
+</sdf>
+```
+
+You can learn more about SDF [here](http://sdformat.org/).
+For this tutorial, it's mostly only important to take note of the `<uri>` tags.
+Following `model://` is the path within the model directory to the mesh.
+
+## Upload to Fuel
+
+Once you have all your content accumulated in a model directory, you're ready to upload to Fuel.
+Click on the `+` button at the top right corner of the page, next to your account button, and then select `Model`.
+
+Typically, you would try to ensure that the `Model name` field matches the `name` tags in both `model.config` and `model.sdf`, and the `Description` field matches its corresponding tag in `model.config`.
+
+For the sake of this tutorial, however, we're going to enter your model name as `Electrical Box Test` to differentiate it from the original. In the `Description` field, write something like `Test model for Importing a Mesh to Fuel tutorial`.
+
+Click the `Add folders` button, or drag and drop the `Electrical Box` folder you unzipped earlier into the labeled area.
+All the files in your model description will be listed there.
+Press `Upload`, and the "Fuel Model Info" page for your model will open.
+
+![Electrical Box Test](https://bitbucket.org/ignitionrobotics/ign-gazebo/raw/875e08edfbe89847c7a9b7f819e7c2d339f6cb14/tutorials/files/mesh_to_fuel/model_info2.png)
```

Comment:
can you add these images to this PR? we can then switch to use the github links to these images.